### DISCUSSION TITLE
Can you configure an global default user-agent?

### DISCUSSION THREAD

**Discussion by 4-FLOSS-Free-Libre-Open-Source-Software** [FreshRSS/FreshRSS#6319] [Q&A] [answered]
Can you choose a default user-agent for every feed?

**Comment by Alkarex** [comment, marked as answer]
It is possible to make a `./constants.local.php` file on the model of:

```php
<?php

define('FRESHRSS_USERAGENT', 'FreshRSS/' . FRESHRSS_VERSION . ' (' . PHP_OS . '; ' . FRESHRSS_WEBSITE . ')');
```

See https://github.com/FreshRSS/FreshRSS/blob/0ffcf41f9371b9321856d2c709cf7bcb4f7c3c60/constants.php#L31
